plugins/processors/regex/converter.go

```diff
@@ -39,8 +78,15 @@ func (c *converter) setup(ct converterType) error {
 				}
 			}
 			if allNamed {
+				log.Infof("%s: Using named-group mode...", ct)
 				c.groups = groups[1:]
+			} else {
+				msg := "Neither 'result_key' nor 'replacement' given with unnamed or mixed groups.\n"
+				msg += "Using explicit, empty replacement!"
```

Comment:
I don't believe we have a history of doing multi-line log messages. Could this either be on a single line, two separate warnings, or an info + warning?

Reply:
I had my doubts on this one too.